src/System.Management.Automation/engine/parser/TypeResolver.cs

```diff
@@ -20,8 +20,7 @@
 using System.Xml;
 using Microsoft.Management.Infrastructure;
 using Microsoft.PowerShell.Commands;
-#if !CORECLR
-// System.DirectoryServices are not in CoreCLR
+#if !UNIX
 using System.DirectoryServices;
 #endif
```

Comment:
Can we sort the using list?

Reply:
We should do a global pass if we want them all sorted

Reply:
I ask only because you have already edited the list.

Reply:
I suppose I could update it just for this file since I'm touching this part anyways.